packages/odyssey-react/.storybook/translations/de.json

```diff
@@ -0,0 +1,28 @@
+{
+  "variant.primary": "Primär",
+  "variant.secondary": "Sekundär",
+  "variant.danger": "Gefahr",
+  "variant.clear": "Löschen",
+  "variant.dismiss": "Verwerfen",
+  "variant.neutral": "Neutral",
+  "variant.success": "Erfolg",
+  "variant.caution": "Vorsicht",
+  "variant.wide": "Breit",
+  "label.fieldLabel": "Feldbezeichnung",
+  "label.statusLabel": "Statuslabel",
+  "label.hint": "Dies ist ein Feldhinweis.",
```

Comment:
I'm with you on this, maintaining translations isn't where I want us to spend our time. I was looking into the possibility of using the google translate API to automatically generate these files for us, which seems suitable long term.

I used [this tool](https://file-translate.com/en/app/json-translate) for our quickly creating the different language JSON files, and while not perfect, gave me a solid starting point.

I don't have a strong opinion and will defer to you, @edburyenegren-okta and @joejezowski-okta on this.

Reply:
There are some benefits to testing with actual Arabic. Aside from the weirdness of RTL English, we'll want to make sure that things like letter- and line-spacing are appropriate for block as well as "connected" or ligatured languages like Arabic. Pseudoloc is very useful for diacritics and other atypical glyphs, but isn't as helpful for design review of visual appearance or reading experience.

That said, I agree that maintaining multiple languages manually seems like a lot of extra work right now. German and Japanese are likely unnecessary. Any concerns that show up in those languages should be covered by either English or Arabic. As @jeffbelser-okta mentioned, concerns around truncation or line-length are pretty minimal given our styling.

My preference would be to maintain an auto-translated Arabic string set. There's still some additional work in maintaining a string set for English, but comes with it's own benefits.

Reply:
Good callouts Shred. I think through a lot of the initial RTL testing with English and the research you've done it's clear to me that we need to see & use the UI we produce. As we start building out more complex UIs it'll be really important to see the implementation in that locale (Arabic).

Reply:
@edburyenegren-okta good thoughts -but are any of us familiar enough with written Arabic to know that our `letter-spacing` or `line-height` would be considered unacceptable? Maybe? I'm not sure its worth the cost given the value but those are two very good specific cases to check for. We should at least spent some time looking into a simple pseudo-localization util ([or even writing our own](https://github.com/ianllewellyn/pseudoizer/blob/master/lib/index.js)) that would perhaps even render text in more ligatured format given the right font. This would allow us to check for the concerns we have but also eliminate the ongoing management of extracted strings.  Thoughts?

Reply:
I think our unfamiliarity with Arabic is one of the bigger reasons to include visual examples. While we don't know if the translations are 💯, @joejezowski-okta knows what to look for visually that would indicate a poor experience for Arabic. 

I'm pretty neutral on approach. [This ticket](https://oktainc.atlassian.net/browse/OKTA-405619) gives good context on what we'd like to be able to test. I'm not an expert in i18n, but I think Arabic is a typical example both because of RTL and also that the script itself covers many language features English doesn't need to account for.

Reply:
oh the UI core pseudo loc project uses [App Localizer](https://www.npmjs.com/package/app-localizer) internally too